config/build.gradle

```diff
@@ -22,7 +22,8 @@ jar {
       'Specification-Title': archiveBaseName,
       'Specification-Version': project.version,
       'Implementation-Title': archiveBaseName,
-      'Implementation-Version': calculateVersion()
+      'Implementation-Version': calculateVersion(),
+      'Commit-Hash': getGitCommitDetails(40).hash
```

Comment:
are we doing this everywhere as a matter of consistency?  It seems like we only currently need this in the `:besu` artifact, but I don't see a reason not to be consistent across all

Reply:
Yes, I updated all of the gradle sub-modules where we're setting Implementation-Version to also set a Commit-Hash for consistency. Currently, it's only used in the besu module